docs/.vuepress/config.js

```diff
@@ -134,14 +153,12 @@ module.exports = {
                 // Custom navbar values
                 nav: [{ text: "Konfiguration", link: "/config/" }],
```

Comment:
Just noticed a bug, while testing:
```suggestion
                nav: [{ text: "Konfiguration", link: "/de-DE/config/" }],
```